Zend/zend_vm_execute.h

```diff
@@ -2504,7 +2508,7 @@ static zend_never_inline ZEND_OPCODE_HANDLER_RET ZEND_FASTCALL zend_dispatch_try
 
 			/* Chain potential exception from wrapping finally block */
 			if (Z_OBJ_P(fast_call)) {
-				if (ex) {
+				if (ex && !zend_is_unwind_exit(ex)) {
```

Comment:
Did you edit the wrong file? I don't see it in the vm_def diff?

Reply:
Other way around: I did not regenerate after dropping a change. Fixed now.